docs/scaffold-a-new-project/README.md

```diff
@@ -0,0 +1,72 @@
+# Scaffold a New Project
+
+Neutrino provides a scaffolding tool to help you quickly jump-start a project. `create-neutrino-project` uses
```

Comment:
"Neutrino can help you quickly start new projects by scaffolding your initial project structure."